transaction/README.md

```diff
@@ -0,0 +1,60 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Geode Transaction Example
+
+This is a simple example that demonstrates the use of [transactions](https://geode.apache.org/docs/guide/11/developing/transactions/working_with_transactions.html)
+to protect consistency during concurrent access and modification of data. Since a region may
+configured across multiple servers and multiple clients may interact with that region independent of
+each other, data integrity relies on synchronization of modifications between all actors.
+
+An example of how data can become inconsistent during concurrent interaction is as follows:
```

Comment:
Technically, this particular issue could be solved with a putIfAbsent since you are only modifying a single key, but I guess it's still a good example of transactions.

Reply:
I'm comfortable with a weaker case for transactions for a more concise example. Do you think this is a showstopper?

Reply:
It's not a showstopper, I get you are trying to make the example simple.